exercises/practice/bank-account/bank_account.odin

```diff
@@ -0,0 +1,36 @@
+package bank_account
+
+TransactionResult :: enum {
+	Success,
+	Account_Not_Open,
+	Account_Already_Open,
+	Invalid_Amount,
+	Not_Enough_Balance,
+}
+
+Account :: struct {} // Implement this struct
+
+open :: proc(self: ^Account) -> TransactionResult {
+	// Implement this procedure.
+	return .Success
```

Comment:
This is where, in other exercises, we have an `.Unimplemented` member of the enum.